testgrid/config/config.yaml

```diff
@@ -2822,6 +2828,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gce-gpu-1-7
   - name: gce-kubeadm-1.7
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-7
+  - name: gce-kubeadm-1.6-on-1.7
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7
```

Comment:
Functionally, I believe our SIG agrees that this feature should be well supported and block the release if broken. But, since this is a brand new test, and those have horrible track records for working on the first try, I'm perfectly willing to leave it out of that testgrid tab until it proves stable.